packages/configs/package.json

```diff
@@ -0,0 +1,36 @@
+{
+	"name": "@pantheon-systems/configs",
+	"version": "0.0.0",
+	"description": "Eslint configs for the @pantheon-systems decoupled SDK npm packages",
+	"license": "GPL-3.0-or-later",
+	"homepage": "https://github.com/pantheon-systems/decoupled-kit-js#readme",
+	"bugs": "https://github.com/pantheon-systems/decoupled-kit-js/issues/new?template=bug-report-template.yml",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/pantheon-systems/decoupled-kit-js"
+	},
+	"author": "@pantheon-systems",
+	"files": [
+		"./index.js",
+		"./react.js",
+		"./prettier.config.js"
+	],
+	"main": "./index.js",
```

Comment:
This looks correct according to the docs, but I'm worried that something about this won't work because the name of the module is not prefixed with "eslint-config". Did you try using `npm link` to test this locally outside of the monorepo?

See https://eslint.org/docs/latest/extend/shareable-configs#creating-a-shareable-config for more info.